lib/chusaku.rb

```diff
@@ -148,5 +150,9 @@ def output_results
         0
       end
     end
+
+    def inspect_defaults(hash)
```

Comment:
What do you think about renaming this to something like `annotate_defaults` to be clearer on its function?

Reply:
Fighting the Complexity, I redesigned the main annotate method, so this one just gone :)